fuji_server/data/longterm_formats.json

```diff
@@ -1,119 +0,0 @@
-{
-  "application/jpg": null,
```

Comment:
These JSON files make no sense now, because all values are null.

There is no domain in file_formats.json/yaml, to populate this. What happened?

Reply:
Yes this can be deleted, further, all the json files mentioned in these comments (longterm_formats, open_formats) are no longer used

Reply:
I removed the files. Still the problem remains, there is no "domain" in file_formats.json/yaml, which the preprocessor tries to pull out.

Reply:
Domain probably meant 'science field' and I think I remember that there were just too little file formats which directly could be related to such a science field.
I think these lines are obsolete since there is no domain any more lets' comment these lines while I check what happened:
``
domain = None
if file.get("domain"):
      domain = file.get("domain")[0]
```

Reply:
I guess this is out of scope of this PR. This PR should only transform json to yaml files.

But really this is curious:

```pycon
>>> preprocessor.get_long_term_file_formats()
{'audio/x-aac': None, 'audio/aac': None, 'image/x-adobe-dng': None, 'audio/x-midi': None, 'sound/aiff': None, 'audio/aiff': None, 'audio/x-rmf': None, 'audio/x-pn-aiff': None, 'audio/x-gsm': None, 'audio/x-aiff': None, 'audio/rmf': None, 'audio/vnd.qcelp': None, 'application/x-troff-msvideo': None, ... }
```

If there is never a value in this dictionary, then it is the wrong data structure? Maybe just use a set?

Reply:
Hmm indeed these xy_file_formats methods should be merged into a single method which returns a dictionary with all file formats and a 'preferrence_reason' (logn term, scientific..) 
Further, I have seen that the variables which consume these dicts in fair_check.py are still class variables (e.g. cls.SCIENCE_FILE_FORMATS) which I don't like very much.

Reply:
I just wanted to point this out. You can open an issue from this conversation.

Reply:
See #527